config/ipam/image_patch.yaml

```diff
@@ -11,15 +11,15 @@ spec:
       - image: quay.io/metal3-io/ip-address-manager:v0.0.6
         name: manager
 ---
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: ipam-controller-manager
-  namespace: capi-webhook-system
-spec:
-  template:
-    spec:
-      containers:
-      # Change the value of image field below to your controller image URL
-      - image: quay.io/metal3-io/ip-address-manager:v0.0.6
-        name: manager
\ No newline at end of file
+# apiVersion: apps/v1
+# kind: Deployment
+# metadata:
+#   name: ipam-controller-manager
+#   namespace: capi-webhook-system
+# spec:
+#   template:
+#     spec:
+#       containers:
+#       # Change the value of image field below to your controller image URL
+#       - image: quay.io/metal3-io/ip-address-manager:v0.0.6
+#         name: manager
```

Comment:
Why is this so?

Reply:
@fmuyassarov the reason is, while testing it locally, I was seeing the following:
```
`make[1]: Leaving directory '/home/ubuntu/tested_repo/capm3'
+ ./hack/tools/bin/kustomize build /home/ubuntu/go/src/github.com/metal3-io/baremetal-operator/config/default
+ sed -i -e s#https://raw.githubusercontent.com/metal3-io/baremetal-operator/master/config/render/capm3.yaml#bmo-components.yaml# config/bmo/kustomization.yaml
+ '[' v1alpha4 == v1alpha3 ']'
+ ./hack/tools/bin/kustomize build /home/ubuntu/tested_repo/metal3-ipam/config/default
+ sed -i -e 's#https://github.com/metal3-io/ip-address-manager/releases/download/v.*/ipam-components.yaml#metal3-ipam-components.yaml#' config/ipam/kustomization.yaml
+ popd
~/tested_repo/capm3 ~/metal3-dev-env
+ make release-manifests
make[1]: Entering directory '/home/ubuntu/tested_repo/capm3'
cd hack/tools; ./install_kustomize.sh
Verifying kustomize version
mkdir -p out/
hack/tools/bin/kustomize build config/default > out/infrastructure-components.yaml
Error: accumulating resources: accumulation err='accumulating resources from '../ipam': '/home/ubuntu/tested_repo/capm3/config/ipam' must resolve to a file': recursed accumulation of path '/home/ubuntu/tested_repo/capm3/config/ipam': no matches for IdId apps_v1_Deployment|capi-webhook-system|ipam-controller-manager; failed to find unique target for patch apps_v1_Deployment|ipam-controller-manager
make[1]: *** [Makefile:415: release-manifests] Error 1
make[1]: Leaving directory '/home/ubuntu/tested_repo/capm3'
make: *** [Makefile:10: launch_mgmt_cluster] Error 2`
```

IMO this is all because of the required changes re: [Required kustomize changes to have a single manager watching all namespaces and answer to webhook calls](https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/book/src/developer/providers/v1alpha3-to-v1alpha4.md#required-kustomize-changes-to-have-a-single-manager-watching-all-namespaces-and-answer-to-webhook-calls). It is suggested to remove the namespace configuration from the `/config/webhook/kustomization.yaml` and meaning that **capi-webhook-system** needs to be removed from kustomization as I did [here](https://github.com/metal3-io/cluster-api-provider-metal3/pull/167/files#diff-ba52e3817845d8a1e6cf9a6d16e0a8afdecea23c686b962dc408e2411e4c01edL1).